api/nakadi-event-bus-api.yaml

```diff
@@ -2215,6 +2269,8 @@ definitions:
           contains at least one of the scopes in this list.
           If no scopes provided then anyone can publish to this event type.
 
+          Usage of write_scopes is deprecated.
```

Comment:
Could you add a "and is ignored" to this sentence, to make this clear? (Similarly for the read_scopes 12 lines below).